site_root/input/resources/profiles/StructureDefinition-kfdrc-no-phi-specimen.json

```diff
@@ -1,10 +1,10 @@
 {
   "resourceType": "StructureDefinition",
-  "id": "biospecimen",
-  "url": "http://fhir.kids-first.io/StructureDefinition/biospecimen",
+  "id": "kfdrc-no-phi-specimen",
+  "url": "http://fhir.kids-first.io/StructureDefinition/kfdrc-no-phi-specimen",
   "version": "0.1.0",
-  "name": "biospecimen",
-  "title": "Biospecimen",
+  "name": "kfdrc-specimen",
+  "title": "Kids First DRC Specimen without Protected Health Information",
```

Comment:
Same comment regarding this needing to be based on the base kfdrc profile that allows phi.

Reply:
Updated and validated!